test/integ/cli/test_cli_asset.py

```diff
@@ -0,0 +1,126 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+Integ tests for the CLI asset commands.
+"""
+import os
+import json
+from click.testing import CliRunner
+import pytest
+import tempfile
+import posixpath
+
+from deadline.client.cli import main
+
+
+class TestSnapshot:
+
+    @pytest.fixture
+    def temp_dir(self):
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            yield tmpdirname
+
+    def test_pass_valid_root_dir(self, temp_dir):
+        """
+        Snapshot with a just a valid root directory, and no other parameters
+        """
+        root_dir = os.path.join(temp_dir, "root_dir")
+        os.makedirs(root_dir)
+        file_path = os.path.join(root_dir, "file.txt")
+        with open(file_path, "w") as f:
+            f.write("test file content")
+
+        runner = CliRunner()
+        runner.invoke(main, ["asset", "snapshot", "--root-dir", root_dir])
+
+        # Check manifest file details to match correct content
+        # since manifest file name is hashed depending on source location, we have to list out manifest
+        manifest_folder_path = os.path.join(root_dir, f"{os.path.basename(root_dir)}_manifests")
+        manifest_file_name = os.listdir(manifest_folder_path)[0]
+        manifest_file_path = os.path.join(manifest_folder_path, manifest_file_name)
+
+        with open(manifest_file_path, "r") as f:
+            manifest_data = json.load(f)
+            assert manifest_data["paths"][0]["path"] == "file.txt"
+            assert (
+                manifest_data["paths"][0]["hash"] == "0741993e50c8bc250cefba3959c81eb8"
```

Comment:
How did you get this hash, could it be a call with the input? i.e. we don't need to update the hash if there is any change to the input.